Symfony/CS/Tests/Fixer/Symfony/NoEmptyLinesAfterPhpdocsFixerTest.php

```diff
@@ -212,4 +212,26 @@ public function testFixesWindowsStyle()
 
         $this->makeTest($expected, $input);
     }
+
+    // empty line between typehinting docs and return statement should be preserved
+    public function testInlineTypehintingDocsBeforeReturn()
+    {
+        $expected = <<<'EOF'
+<?php
+
+function parseTag($tag)
+{
+    $tagClass = get_class($tag);
+
+    if ('phpDocumentor\Reflection\DocBlock\Tag\VarTag' === $tagClass) {
+        /** @var DocBlock\Tag\VarTag $tag */
```

Comment:
On second thought, this fixer is about removing empty lines after docblock, so it should just remove this one. Having newlines before returns is out of scope for this fixer. If people use docblocks in an odd way, its up to them to fix their docblock usage. Also when running this fixer together with docblocktocmment fixer, this docblock will be converted to normal comments.